src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesDataController.java

```diff
@@ -109,4 +112,39 @@ private ResponseEntity<OverseasEntityDataApi> getOverseasEntityDataResponse(
         
         return ResponseEntity.ok(overseasEntityDataApi);
     }
+
+    @GetMapping("/beneficial-owners")
```

Comment:
We'll need to update the `routes.yml` and `InterceptorConfig.java` files with the new path so that we can access the endpoint

Reply:
Added